public/static/locales/en/common.json

```diff
@@ -232,12 +236,12 @@
   "Other Details": "Other Details",
   "Business Type": "Business Type",
   "Water Supply": "Water Supply",
-  "Contact Details": "Contact Details",
+  "Contact Details": "Contact details",
```

Comment:
Why are these two changed to lower case for the second word and not the rest? Shouldn't the key also match the value?